.github/workflows/smoke.yml

```diff
@@ -51,7 +51,11 @@ jobs:
     - name: Smoke Test
       shell: bash
       run: |
-        cd tests && ./main.sh smoke
+        cd tests
```

Comment:
Honestly I think we can do this better so we're not left holding the bag with Github changing stuff.

My proposal is to make a new directory in the tests and export the env variable JUJU_DATA to that dir.

We then don't need to rely on .local/share

Reply:
Yep I think setting JUJU_DATA is a good idea, to a directory that we can control.

Reply:
So I set JUJU_DATA and now I'm [getting](https://github.com/juju/juju/runs/6317983150?check_suite_focus=true#step:8:13)
```
ERROR Permission denied, are you in the lxd group?
```
Definitely some kind of weird permissions problem

Reply:
Tried to setup LXD as per [here](https://juju.is/docs/olm/lxd#heading--configure-lxd), let's see if it helps.

Reply:
yeah that's a different problem now. You may want to look at `mktemp -d` for creating the JUJU_DATA directory.